ManagementPacks/Community.DataOnDemand.Unix/Scripts/ExtractUserFromProfile.ps1

```diff
@@ -0,0 +1,24 @@
+﻿# Extract username from a Unix/Linux run as profile prior to passing to the Microsoft.SystemCenter.WSManagement.Invoker
+Param(
+	$user
+)
+
+$scomAPI = New-Object -ComObject "MOM.ScriptAPI"
+$propertyBag = $scomAPI.CreatePropertyBag()
+
+if ([string]::IsNullOrEmpty($user))
+{
+	$propertyBag.AddValue("User", "")	
+}
+elseif ($user -match '<UserId>.+</UserId>')
+{
+	$modifiedUser = $user -replace '.*<UserId>([^<]+)</UserId>.*','$1'
```

Comment:
Is there any possibility of [HTML character escapes](https://www.w3.org/International/questions/qa-escapes) being present in here - and, if so, do we need to do anything about them?